[Search] [Playground] present message when no documents are returned (#183219)
elastic/kibana: docs/playground/playground-query.asciidoc

Retrievers make it easier to compose and test different retrieval strategies in your search pipelines. 
====
// TODO: uncomment and add to note once following page is live

> //Refer to {ref}/retrievers-overview.html[documentation] for a high level overview of retrievers.

Reviewer: ```suggestion
Refer to {ref}/retrievers-overview.html[documentation] for a high level overview of retrievers.
```

🚗 uncommenting

> [[playground-hidden-fields]]

Reviewer: ```suggestion
[float]
[[playground-hidden-fields]]
```

asciidoc weirdness: without `[float]` doc will render on new page

Reviewer: should fix ci

=== Hidden fields

> We present a simplified view of choosing fields to search on. The query editor presents fields which are ideal for the user to search on.

Reviewer: ```suggestion
The query editor shows fields which make sense for the user to search on, but not all fields in your documents are visible from the editor.
```

> Available Field types:

Reviewer: ```suggestion
Available field types:

```

- Semantic fields like `sparse_vector` or `dense_vector` fields where the embeddings have been created from a inference processor
- `text` fields

> hidden Field Types:

Reviewer: ```suggestion
Hidden Field Types:

```
